docs/conf.py

```diff
@@ -16,19 +16,36 @@
 # import sys
 # sys.path.insert(0, os.path.abspath('..'))
 import sphinx_rtd_theme
-
+from builtins import str
+import re
+import subprocess
+import os
 
 # -- Project information -----------------------------------------------------
 
 project = u'NVIDIA DALI'
 copyright = u'2018, NVIDIA Corporation'
 author = u'NVIDIA Corporation'
 
+version_long = u'0.0.0'
+with open("../VERSION") as f:
+    version_long = f.readline()
+
+version_short = re.match('^[\d]+\.[\d]+', version_long).group(0)
+
+git_sha = ""
+try:
+    git_sha = subprocess.check_output(["git", "log", "--pretty=format:'%h'", "-n1"]).decode('ascii').replace("'","").strip()
```

Comment:
This is wrong - it will give you the commit hash of the commit used to build the docs, which is not the same as the commit on GitHub.

Reply:
It will be like that once we get the CI automation sorted out.  I'm ok with tabling that issue until then.

Reply:
I switched order a bit. First, it tries to obtain SHA from env, if fails it gets from the source. In our CI we can provide SHA as an env variable in the way we want i.e. 'git merge-base'.
I prefer to keep this 'git log' as if someone wants to generate docs he will do it form his repo.
How about that.

Reply:
ok, makes sense.